plugins/hls-overloaded-record-dot-plugin/test/Main.hs

```diff
@@ -28,13 +42,36 @@ test = testGroup "overloaded-record-dot"
     mkTest "MultilineCase" "MultilineCase" "name" 10 7 12 15,
     mkTest "Multiline" "Multiline" "name" 10 7 11 15,
     mkTest "MultilineExpanded" "MultilineExpanded" "owner" 28 8 28 19
-  ]
+  ],
+  testGroup
+  "with Resolve"
+  [ mkResolveTest "Simple" "Simple" "name" 10 7 10 15,
```

Comment:
I think the idea is that we use the same expected files for both kinds of test since they should have the same result?